.travis.yml

```diff
@@ -11,5 +11,5 @@ deploy:
   on:
     all_branches: true
     tags: true
-    node: '7'
+    node: '8'
```

Comment:
If you remove this line, it will use the node version specified in the root of this file